app/src/main/java/com/example/habitsmasher/User.java

```diff
@@ -16,8 +15,8 @@
     private String _password;
     private String _email;
     private String _id;
-    private final ArrayList<String> _followerList = new ArrayList<>();
-    private final ArrayList<String> _followingList = new ArrayList<>();
+    private final ArrayList<String> _followers = new ArrayList<>();
+    private final ArrayList<String> _following = new ArrayList<>();
```

Comment:
Renamed for clarity, and avoiding _Hungarian Notation_ (look it up if you're curious!)